README.md

```diff
@@ -50,7 +50,7 @@ for issue in Issue.get_content_issues('issue_header'):
 
 # Contributing
 
-Review the [Contributing Guidelines](CONTRIBUTING.md) for information on how to contribute to this project.
+Review the [Contributing Guidelines](https://github.com/Python-Community-News/GH-Issues/blob/main/CONTRIBUTING) for information on how to contribute to this project.
```

Comment:
The Contributing Guidelines are shared amongst the org file

```suggestion
Review the [Contributing Guidelines](https://github.com/Python-Community-News/.github/blob/main/CONTRIBUTING) for information on how to contribute to this project.
```

Reply:
I have no idea how I did that... I am almost impressed with how many times I have ruined this PR. Thanks for your patience.